python/ray/tune/trainable/session.py

```diff
@@ -207,6 +207,15 @@ def run_me(config):
             )
 
         return _session(_metric, **kwargs)
+    else:
+        # Backwards compatibility
+        from ray import train
+
+        report_dict = kwargs
+        if _metric is not None:
+            report_dict["_metric"] = _metric
+
+        train.report(report_dict)
```

Comment:
should we just force users to switch to train.report? otherwise we'll have to force users to do another migration post 2.7

Reply:
I'll add a deprecation warning for now. Let's check if we can hard deprecate in a follow-up